test/unit/base.test.js

```diff
@@ -37,6 +38,22 @@ describe('CONVERT FUNCTION TESTS ', function() {
         expect(conversionResult.output[0].data).to.have.property('info');
         expect(conversionResult.output[0].data).to.have.property('item');
 
+        done();
+      });
+    });
+    it('[Github: 102]- Should generate collection description with only given fields in info object ' +
+      specPath2, function(done) {
+      Converter.convert({ type: 'file', data: specPath2 }, { schemaFaker: true }, (err, conversionResult) => {
+        let description = conversionResult.output[0].data.info.description;
```

Comment:
Should check for info to be present before trying to access property description of it @Dhroov7